src/commands/readme.ts

```diff
@@ -32,6 +32,7 @@ Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.
     aliases: Flags.boolean({allowNo: true, default: true, description: 'include aliases in the command list'}),
     dir: Flags.string({default: 'docs', description: 'output directory for multi docs', required: true}),
     multi: Flags.boolean({description: 'create a different markdown page for each topic'}),
+    readmePath: Flags.string({default: 'README.md', description: 'Path to the README file.', required: true}),
```

Comment:
can you rename this to `readme-path` to stay consistent with our naming convention? Other than that, this change looks great!